src/common/legacy_config_opts.h

```diff
@@ -20,7 +20,6 @@ OPTION(cluster_addr, OPT_ADDR)
 OPTION(public_network, OPT_STR)
 OPTION(cluster_network, OPT_STR)
 OPTION(lockdep, OPT_BOOL)
-OPTION(lockdep_force_backtrace, OPT_BOOL) // always gather current backtrace at every lock
```

Comment:
the description is lost, please add `set_description()` in `options.cc` to corresponding options.